src/main/java/pl/edu/mimuw/usos/Faculty/IndexCounter.java

```diff
@@ -0,0 +1,10 @@
+package pl.edu.mimuw.usos.Faculty;
+
+public class IndexCounter {
+  private static Integer indexCounter = 0;
+
```

Comment:
`private IndexCounter() {}` uniemożliwi nam tworzenie instancji tej klasy  tym razem o to nam chodzi raczej